test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelperSampleTest.cs

```diff
@@ -18,44 +22,110 @@ public TagHelperSampleTest(MvcTestFixture<TagHelperSample.Web.Startup> fixture)
 
         public HttpClient Client { get; }
 
-        public static TheoryData<string> PathData
+        [Fact]
```

Comment:
The site is stateful, sure. But you're only checking the status code of the response. How much connection is there between the requests?

What you've written in this test method is a story, not a test or two of the controller's actions.

My recommendation is to call whatever `Create` action is necessary to set the groundwork for `Edit` actions in one `[Theory]`'s `// Arrange` and test the `Create` action itself in another `[Theory]`.